### PR TITLE
Ldt 126

### PR DESCRIPTION
**Pull request by architolk**

### CODE REVIEW

File: vocabulary/MAPPING.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+|elmo 1.0 term|elmo 2.0 term|Comment|Realised
+|-------------|-------------|---------|------------
+`elmo1:Assertion`|||No
+`elmo1:Container`|`elmo2:Representation`|with regard to the URI matching|No
+`elmo1:Container`|`elmo2:Transaction`|with regard to the processing|No
+`elmo1:VersionContainer`|`elmo2:VersionContainer`|see `elmo1:Container`|No
+`elmo1:Fragment`|`elmo2:Fragment`||Legacy
+`elmo1:Form`|||No
+`elmo1:GeoLocator`|||No
+`elmo1:Part`|`elmo2:Representation`|No need for specific class `elmo1:Part` any more|Yes
```
Comment (lwortel) [inline vocabulary/MAPPING.md]: anymore

File: vocabulary/MAPPING.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+`elmo1:Production`|`elmo2:Representation`|with regard to the URI matching|No
+`elmo1:Production`|`elmo2:Transaction`|with regard to the processing|No
+`elmo1:Query`|`elmo2:InformationProduct`|The need for a `elmo:Query` class is not necessary any more with the availability of `elmo2:InformationProduct`|Yes
```
Comment (lwortel) [inline vocabulary/MAPPING.md]: The `elmo:Query` is not necessary anymore with...
OF
The need for `elmo:Query` is gone with...

File: vocabulary/MAPPING.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+## Properties
+
+### `elmo1:Representation` and `elmo1:Part`
+
+|elmo 1.0 term|elmo 2.0 term|Comment|Realised
+|-------------|-------------|-------|--------
+||`elmo2:informationProduct`|Links an `elmo2:Representation` to its `elmo2:InformationProduct`|Yes
+|`rdfs:label`|`rdfs:label`||Yes
+|`elmo1:accept`|||No
+|`elmo1:appearance`|`elmo2:appearance`|`elmo2:appearance maps` to instances of `elmo2:Appearance`, not to classes as does `elmo1:apearance`|Yes
```
Comment (lwortel) [inline vocabulary/MAPPING.md]: `elmo2:appearance` maps
`elmo1:appearance` ipv apearance

File: vocabulary/MAPPING.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+|`elmo1:applies-to`|`elmo2:appliesTo`||No
+|`elmo1:container`|||No
+|`elmo1:contains`|`elmo2:contains`||Yes
+|`elmo1:data`|`elmo2:data`|Static data is net yet supported. You might use an IP dat delivers only static content|No
```
Comment (lwortel) [inline vocabulary/MAPPING.md]: that ipv dat

File: vocabulary/MAPPING.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+### `elmo1:Fragment`
+
+|elmo 1.0 term|elmo 2.0 term|Comment|Realised
+|-------------|-------------|-------|--------
+|`rdfs:label`|`rdfs:label`||Yes
+|`elmo1:applies-to`|`elmo2:appliesTo`||Yes
+|`elmo1:backmap`|`elmo2:backmap`||Legacy, not yet
+|`elmo1:index`|`elmo2:index`||Legacy, not yet
+|`elmo1:appearance`|`elmo2:apperance`||Legacy, not yet
```
Comment (lwortel) [inline vocabulary/MAPPING.md]: appearance ipv apperance

File: vocabulary/MAPPING.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+### `elmo1:Fragment` (in case of plain json formatting)
+
+|elmo 1.0 term|elmo 2.0 term|Comment|Realised
+|-------------|-------------|-------|--------
+|`elmo1:name`|||No
+|`elmo1:path`|||No
+
+### `elmo1:Container` and `elmo1:VersionContainer`
+Translates to `elmo2:Transaction`, except when explictly stated.
```
Comment (lwortel) [inline vocabulary/MAPPING.md]: Except when explicitly stated otherwise?

File: vocabulary/MAPPING.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+|-------------|-------------|-------|--------
+|`elmo1:contains`|`elmo2:contains`|As part of the elmo2:Representation|Yes
+|`elmo1:queryForm`|||No
+
+### `elmo:Scene`
+|elmo 1.0 term|elmo 2.0 term|Comment|Realised
+|-------------|-------------|-------|--------
+|`rdfs:label`||||No
+|`elmo:index`||No need for index: postQueries is an ordered list|No
+|`elmo1:query`||A item from the `elmo2:postQueries` list as part of an `elmo2:Transaction`|No
```
Comment (ingridthurlings) [inline vocabulary/MAPPING.md]: An item ipv a item

**Comment by ingridthurlings** [comment]
Lars was sneller met de typfouten corrigeren ;) verder oké

**Pull request merged**